src/cli/index.ts

```diff
@@ -101,12 +101,15 @@ async function verify(bundlePath: string, artifactPath: string) {
   const bundleFile = fs.readFileSync(bundlePath);
   const bundle = JSON.parse(bundleFile.toString('utf-8'));
 
-  const result = await sigstore.verify(bundle, payload);
-
-  if (result) {
+  try {
+    await sigstore.verify(bundle, payload);
     console.error('Verified OK');
-  } else {
-    throw 'Signature verification failed';
+  } catch (e) {
+    console.error('Verification failed');
+    if (e instanceof Error) {
```

Comment:
Aren't all error's instances of the `Error` class? Or what is this guard protecting against?

Reply:
In JS you can technically `throw` anything, so if I want to access `message` here TypeScript forces me to first prove it's an `Error`.